### PR TITLE
Dispersed Volume and Distributed-Dispersed Volume creation is added

### PR DESCRIPTION
**Pull request by nishith-vihar**
Dispersed and distributed-dispersed volume type creation is added and
according values of bricks are changed in config file for a 3 server
setup and some pylint fixes are are made. the new volume types are added
in the sample test case for volume creation.

Fixes: #265
Signed-off-by: Nishith Vihar Sakinala <nsakinal@redhat.com>

### CODE REVIEW

**Comment by nishith-vihar** [comment]
Screenshot from 2021-05-07 12-24-09

File: common/ops/gluster_ops/volume_ops.py [hunk 1]
```diff
@@ -107,34 +106,47 @@ def volume_create(self, volname: str, node: str, conf_hash: dict,
...
+                cmd = (f"gluster volume create {volname} replica {conf_hash['replica_count']} "
+                       f"arbiter {conf_hash['arbiter_count']}{brick_cmd}")
+            # replicated vol
             else:
-                cmd = (f"gluster volume create {volname} replica {conf_hash['replica_count']}{brick_cmd}")
+                cmd = (f"gluster volume create {volname} replica {conf_hash['replica_count']}"
+                       f"{brick_cmd}")
+        # dispersed vol and distributed-dispersed vol
+        elif "disperse_count" in conf_hash:
+            cmd = (f"gluster volume create {volname} disperse {mul_fac}"
```
Comment (aujjwal-redhat) [inline common/ops/gluster_ops/volume_ops.py:147]: There should be a space at the end.

**Review by aujjwal-redhat — CHANGES_REQUESTED** [review]

File: common/ops/gluster_ops/volume_ops.py (hunk 1 above)
Comment (nishith-vihar) [reply common/ops/gluster_ops/volume_ops.py:147]: Ack

**Review by aujjwal-redhat — APPROVED** [review]
LGTM

**Review by srijan-sivakumar — APPROVED** [review]
LGTM. Finally, one small milestone. We have the capability to create all volumes when we follow the config :)

**Pull request merged**